client/lib/abtest/active-tests.js

```diff
@@ -21,14 +21,6 @@ module.exports = {
 		defaultVariation: 'noNotice',
 		allowAnyLocale: true
 	},
-	plansPageBusinessAATest: {
```

Comment:
@scruffian @stephanethomas yes, you can kill it, will use the events collected so far, no need to keep it.